README.md

```diff
@@ -1,2 +1,10 @@
 # FORMS
 HTML forms
+## the live site is at https://ashah15.github.io/FORMS/
+
+### Description: 
+Created a sign up form 
+
+#### Authors:
+[Asha] (https://github.com/Ashah15)
+[Micheals] (https://github.com/MarvellousUbani)
```

Comment:
`AWESOME`

Nice job creating a `README.md` file, this gives the project more description of what it does and how it does it.

-----

`SUGGESTION`

- Add the assignment link, the project specification, screenshots and other pieces of information that would describe the project.
- When linking a file using `markdown` don't put space between the square brackets and the parenthesis eg. 
`[Asha](https://github.com/Ashah15)` would appear as [Asha](https://github.com/Ashah15) you can learn more about markdown syntax [here](https://www.markdownguide.org/cheat-sheet/)